core/domain/draft_upgrade_services.py

```diff
@@ -90,6 +90,22 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v34_dict_to_v35_dict(cls, draft_change_list):
+        """Converts draft change list from state version 34 to 35. State
+        version 35 add a contentDescription to ImageWithRegion in
+        ImageClickInput interaction, for which there should be no
+        changes to drafts.
```

Comment:
Hmm, is this true? What if there is a draft where the customization args of imageregion input interaction is changed (using STATE_PROPERTY_INTERACTION_CUST_ARGS id in the changelist), wouldn't you have to modify the `new_value` field in that?